### PR TITLE
feat: refactor swift templates

### PR DESCRIPTION
**Pull request by everly-gif**
refactor swift templates

### CODE REVIEW

File: templates/swift/base/params.twig [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+          with: {% if method.parameters.path %}{% if parameter.isGlobal %}self.{% endif %}{{ parameter.name | caseCamel | escapeKeyword }}{% else %}""{% endif %}
+        )
+{% endfor %}
+{% if method.parameters.query or method.parameters.body or method.parameters.formData or _self.methodNeedsSecurityParameters(method) %}
+        {% if 'multipart/form-data' in method.consumes %}var{% else %}let{% endif %} params: [String: Any?] = [
+{% else %}
+        let params: [String: Any?] = [:]
+{% endif %}
+{% for parameter in method.parameters.query | merge(method.parameters.body) %}
+            "{{ parameter.name }}": {% if parameter.isGlobal %}self.{% endif %}{{ parameter.name | caseCamel | escapeKeyword }}{% if not loop.last or _self.methodNeedsSecurityParameters(method) %},{% endif %}
```
Comment (abnegate) [inline templates/swift/base/params.twig:17]: Please add a newline after this one to keep each parameter on their own line

File: templates/swift/base/params.twig [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+{% endfor %}
+{% if _self.methodNeedsSecurityParameters(method) %}
+{% for node in method.security %}
+{% for key,header in node|keys %}
+            "{{header|caseLower}}": client.config["{{header|caseLower}}"]{% if not loop.last %},{% endif %}
```
Comment (abnegate) [inline templates/swift/base/params.twig:22]: Please add a newline after this one to keep each parameter on their own line

File: templates/swift/base/params.twig (hunk 1 above)
Comment (everly-gif) [reply templates/swift/base/params.twig:17]: made the changes

File: templates/swift/base/params.twig (hunk 2 above)
Comment (everly-gif) [reply templates/swift/base/params.twig:23]: made the changes, let me know if its sufficient

**Pull request merged**